languagetool-core/src/main/java/org/languagetool/rules/GRPCRule.java

```diff
@@ -93,6 +94,9 @@ public abstract class GRPCRule extends RemoteRule {
   private static final int DEFAULT_BATCH_SIZE = 8;
   public static final String WHITESPACE_REGEX = "[\u00a0\u202f\ufeff\ufffd]";
   private static final String DEFAULT_DESCRIPTION = "INTERNAL - dynamically loaded rule supported by remote server";
+  /*TODO Delete this temporal fix as this is for speeding up execution for too long sentences*/
+  private final int maxSentenceLength = serviceConfiguration.getOptions().get("maxSentenceLength") == null ? Integer.MAX_VALUE : Integer.valueOf(serviceConfiguration.getOptions().get("maxSentenceLength"));
```

Comment:
Please move the initialization to the constructor, where other options are initialized.